Sources/client-sdk-swift/config/transport/GrpcConfigurationProtocol.swift

```diff
@@ -0,0 +1,4 @@
+protocol GrpcConfigurationProtocol {
+    var deadlineMillis: Int { get }
```

Comment:
In languages that offer one, we've been preferring to use types that represent durations instead of ints. Swift has [just such a `Duration` type](https://developer.apple.com/documentation/swift/duration?changes=_3) that we should use here.

Reply:
on the topic of minimum supported versions, it looks like Duration requires iOS 16.0+ 🙈

Reply:
AAAAARGGGGH

Reply:
OK, can we use [`TimeInterval`](https://developer.apple.com/documentation/foundation/timeinterval) instead then? It's just a type alias to `Double`: "always specified in seconds; it yields sub-millisecond precision over a range of 10,000 years." We should be safe with it's version requirements: iOS 2+.